src/main.cpp

```diff
@@ -110,6 +189,38 @@ int main(int argc, char* argv[])
 
       return 0;
    }
+   if (app_options.user_string_dict().length() > 0) {
+      std::string path =
+         std::regex_replace(app_options.user_string_dict(), std::regex("\\\\"), "\\");
+      path = std::regex_replace(path, std::regex("\""), "");
+      if (fs::exists(path))
+         read_dictionary(path.c_str());
+      else {
+         std::cout << "Error: file '"s << path << "' does not exist\n";
+
+         return 0;
+      }
+   }
```

Comment:
The arguments are processed by app_options which applies 'std::quoted'; here I'm removing the quotes and reducing the '\\' to '\'.

I don't really understand why you want to apply quoted stuff in the first place 
Perhaps you can suggest a better solution for that?

Reply:
Ah I see, are you referring to this use of `std::quoted` or something else? https://github.com/SleepKiller/swbf-unmunge/blob/1d6d0d03dccb0be00b8460bf84378019ac7ec138/src/app_options.cpp#L27
When used on an input stream it should be stripping away quotes (not adding them) so that arguments can be passed in like `-file "C:\Test Folder\file.lvl"`.

I'm still unsure why `\\` is getting replaced with `\` though. If the tool gets passed a path with `\\` in it and the OS rejects it then isn't all the tool needs to do is report the error?

Reply:
After further inspection, I resolved the backslash and quote issue in 'App_options.cpp' by instead doing:
```
{"-string_dict"s, [this](Istr& istr) { _user_string_dict = read_file_path(istr); },
```